### PR TITLE
4295 send emails

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/business/useCases/docketEntry/fileDocketEntryInteractor.js [hunk 1]
```diff
@@ -173,6 +173,15 @@ exports.fileDocketEntryInteractor = async ({
           { applicationContext },
         ),
       );
+
+      const servedParties = aggregatePartiesForService(caseEntity);
+
+      await applicationContext.getUseCaseHelpers().sendServedPartiesEmails({
```
Comment (rachaelparris) [inline shared/src/business/useCases/docketEntry/fileDocketEntryInteractor.js]: I'm not sure that this is the right interactor to call this in. This interactor could be called when we're just saving a docket entry, not saving and serving, right?

Comment (codyseibert) [reply shared/src/business/useCases/docketEntry/fileDocketEntryInteractor.js]: This interactor is called when you do a Save and Serve.

Save and serve does a request to POST@/case-documents/*/docket-entry

which seems to call the `fileDocketEntryToCaseLambda` method

Comment (rachaelparris) [reply shared/src/business/useCases/docketEntry/fileDocketEntryInteractor.js]: Or maybe this just needs to move further up? Inside `if (metadata.isFileAttached && !isSavingForLater) {`? Although that isn't creating a paper service document, so I'm not sure this is the right place either.

Comment (rachaelparris) [reply shared/src/business/useCases/docketEntry/fileDocketEntryInteractor.js]: It looks like this interactor is also called when you do a "Save for Later", and I'm guessing you do not want to serve it then.

**Review by rachaelparris — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**